packages-native/animation/src/Animation.tsx

```diff
@@ -0,0 +1,77 @@
+import { Component, ReactNode, createElement } from "react";
+import { flattenStyles } from "@native-mobile-resources/util-widgets";
+import { Animation as AnimationType, View, Easing } from "react-native-animatable";
+
+import { AnimationProps } from "../typings/AnimationProps";
+import { defaultAnimationStyle, AnimationStyle } from "./ui/Styles";
+
+export type Props = AnimationProps<AnimationStyle>;
+type Direction = "normal" | "reverse" | "alternate" | "alternate-reverse";
```

Comment:
Why do we create our own type's instead of using the libs ? If user changes the core functionality along with these type's it would be hard to maintain..

Reply:
I opened a PR for the typings to be in the repo
https://github.com/oblador/react-native-animatable/pull/287
For now quickly the source